MAUI/DataForm/getting-started.md

```diff
@@ -81,6 +65,40 @@ The [Syncfusion.Maui.Core](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.C
 {% endhighlight %}
 {% endtabs %}
 
+## Step 4: Add a Basic DataForm view.
+
+1. To initialize the control, import the DataForm namespace into your code.
+2. Initialize [SfDataForm](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataForm.SfDataForm.html)
+
+{% tabs %}
+{% highlight XAML %}
+
+<ContentPage   
+            
+        xmlns:dataForm="clr-namespace:Syncfusion.Maui.DataForm;assembly=Syncfusion.Maui.DataForm">
+
+        <dataForm:SfDataForm />
+</ContentPage>
+
+{% endhighlight %}
+{% highlight C# %}
+
+using Syncfusion.Maui.DataForm;
+. . .
+
+public partial class MainPage : ContentPage
+{
+    public MainPage()
+    {
+        InitializeComponent();
+        SfDataForm dataForm = new SfDataForm();
+        this.Content = dataForm;
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
 ## Creating data object
```

Comment:
after step 5, change - ## Step 6: Set data object to data form

Reply:
Added the changes